Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsViewModel.kt

```diff
@@ -58,6 +62,14 @@ class RecoveryCertificateDetailsViewModel @AssistedInject constructor(
         recoveryCertificateRepository.acknowledgeState(containerId)
     }
 
+    fun onExport() {
+        if (recoveryCertificate.value?.certificateCountry != DccCountry("DE").displayName()) {
```

Comment:
What about something like
```kotlin
fun CwaCovidCertificate.canBeExported() = certificateCountry == DccCountry("DE").displayName() 
```
to don't repeat the implementation in all ViewModels?